mock-registry/lib/index.js

```diff
@@ -66,14 +90,14 @@ class MockRegistry {
       // find mistakes quicker instead of waiting for the entire test to end
       t.afterEach((t) => {
         t.strictSame(server.pendingMocks(), [], 'no pending mocks after each')
-        t.strictSame(server.activeMocks(), [], 'no active mocks after each')
```

Comment:
According to nock's docs we almost certainly didn't want this, and the persistent mocks interceptor makes this fail.